languages/en.json

```diff
@@ -2,16 +2,16 @@
     "mcc": {
         "appTitle": "Module Compatibility Checker",
         "statuses": {
-            "x": "Package does not function at all or prevents the world loading",
-            "o": "Package functions with some errors",
-            "b": "Blocked from being tested",
-            "g": "Package works without issue or an updated version is available",
-            "n": "No testing necessary",
-            "a": "Archived - To be archived or no longer available",
-            "u": "Unknown status - Package may not be in the official listing"
+            "x": "Package does not function at all or prevents the world loading.",
+            "o": "Package functions with some errors.",
+            "b": "Blocked from being tested.",
+            "g": "Package works without issue or an updated version is available.",
+            "n": "No testing necessary.",
+            "a": "Archived - To be archived or no longer available.",
+            "u": "Unknown status - Package may not be in the official listing."
         },
         "loading": "Loading...",
-        "selectVersion": "Check compatibility for",
+        "selectVersion": "Check compatibility for:",
```

Comment:
Unnecessary punctuation: these aren't meant to be sentences

Reply:
You should just change things how you see fit... But of course run it and see what it looks like.  Those are used as tooltips and in the custom pie graph legend. Looks like sentences to me. But again.. Change whatever seems to make sense. Hopefully the work is sound and solid and a good tool is released to the community.